examples/create_letters_from_csv_async/letters.py

```diff
@@ -0,0 +1,171 @@
+from __future__ import print_function
+# usage:
+#   python letters.py input.csv
+#   logs to csv's in the output directory
+#
+
+import sys
+try:
+    import concurrent.futures
+except ImportError:
+    print("This script requires Python 3.2 or later")
```

Comment:
Since we still support some versions of Python 2, should we leave also leave a link to the Python 2 [futures](https://pypi.org/project/futures/) package if they choose to stick with Python 2? Up to you though 😛

Reply:
good point, i wouldn't be surprised if a lot of ppl are still on pythong 2